elemental-iso-build

```diff
@@ -11,7 +13,7 @@ build() {
     echo "Using custom registration from $CLOUD_CONFIG_FILE"
     cp "$CLOUD_CONFIG_FILE" "$TEMPDIR"/iso/config
   fi
-  curl https://raw.githubusercontent.com/rancher/elemental/main/Dockerfile.iso -fsSL -o "$TEMPDIR"/Dockerfile
+  cp "${BASEDIR}/Dockerfile.iso" "${TEMPDIR}/Dockerfile"
   curl https://raw.githubusercontent.com/rancher/elemental/main/iso/grub.cfg -fsSL -o "$TEMPDIR"/iso/grub.cfg
```

Comment:
This should not be needed as the elemental-cli already includes the grub.cfg and no download should be needed. However for now I kept it since it includes some weird logic to set a version in a grubenv, I have no idea why we need this, I'd drop it. However I prefer doing it in a separate PR, this is just to fix the broken build. Cosmetic details on grub setup specific to this repository will come later.

Reply:
The grub.cfg file was in there since the start and was copied into the image always.

The version stuff can be dropped, this was for dev purposes and should have no bearing in the final code. In fact we do the same on OBS with an empty version IIRC.